cdisc_rules_engine/utilities/rule_processor.py

```diff
@@ -179,19 +181,29 @@ def rule_applies_to_class(self, rule, file_path, datasets: List[dict], domain: s
         if included_classes:
             if ALL_KEYWORD in included_classes:
                 return True
-            dataset = self.data_service.get_dataset(dataset_name=file_path)
+            variables = PandasDataset(
+                pd.DataFrame(
+                    columns=self.data_service.get_variables_metadata(
+                        dataset_name=file_path
+                    ).variable_name
```

Comment:
When I run a validation, I am getting an Error: 'PandasDataset' object has no attribute 'variable_name'. Error message: 'PandasDataset' object has no attribute 'variable_name'

I think it is caused by this line or 203

Reply:
@gerrycampion I have corrected this and there is no error now, it is data.variable_names in the new dataset structure.  Sorry.